sound/soc/sof/ipc4-topology.h

```diff
@@ -57,6 +57,7 @@
 #define SOF_IPC4_INVALID_NODE_ID	0xffffffff
 
 #define INIT_PAYLOAD_WITH_OUTPUT_FMT	BIT(0)
+#define INIT_PAYLOAD_WITH_BASE_CFG_EXT	BIT(1)
```

Comment:
why is this part for the topology definitions?

Reply:
because we dont have anything to indicate this in the manbifest today, It might change tomorrow but until then this needs to come from topology

Reply:
that's completely broken, I can't sign-off on this. @lgirdwood @mwasko we need clearer definitions on what each module requires, and we need them reported to the kernel. We can't put module-related stuff in the topology, that's just wrong and error prone.